pkg/ebpf/common_test.go

```diff
@@ -25,6 +25,14 @@ func TestUbuntu44119NotSupported(t *testing.T) {
 	}
 }
 
+func TestLinuxAWSPreceding441060NotSupported(t *testing.T) {
+	for i := uint32(120); i < 128; i++ {
+		ok, err := verifyOSVersion(linuxKernelVersionCode(4, 4, i), "Linux-4.4.0-1060-aws", nil)
+		assert.False(t, ok)
+		assert.NotEmpty(t, err)
```

Comment:
FYI there is an `assert.Error()` func for this.

Reply:
Does `assert.Error()` work with `string`?

Reply:
To be fair, the variable `err` here is misleading, it should be called `message` or something instead, in golang `err` usually represents type `error`.

Reply:
Makes sense - and good catch 💯 I'll change the involved tests accordingly